rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```diff
@@ -72,6 +72,7 @@
     private final BridgeSupportFactory bridgeSupportFactory;
     private final byte chainId;
     private final long gasEstimationCap;
+    private final long gasCap;
```

Comment:
minor thing.. `gasCallCap` ? - just to distinguish it from `gasEstimationCap `

Reply:
Done